src/Stratis.Features.FederatedPeg/NetworkHelpers/IPEndPointComparer.cs

```diff
@@ -1,21 +1,39 @@
 ﻿using System.Collections.Generic;
 using System.Net;
+using Stratis.Bitcoin.Utilities.Extensions;
 
 namespace Stratis.Features.FederatedPeg.NetworkHelpers
 {
-    public class IPAddressComparer : IEqualityComparer<IPAddress>
+    public class IPEndPointComparer : IComparer<IPEndPoint>
```

Comment:
Nice, can you add 1 test that creates 2 IP endpoints with the exact same parameters and proves equality? Can be tiny!

Reply:
Done.